lib/trento_web/controllers/session_controller.ex

```diff
@@ -271,5 +271,6 @@ defmodule TrentoWeb.SessionController do
 
   defp maybe_validate_totp(_, _), do: {:error, :totp_code_missing}
 
-  defp idp_redirect_uri, do: Application.fetch_env!(:trento, :oidc)[:callback_url]
+  defp idp_redirect_uri("oidc_local"), do: Application.fetch_env!(:trento, :oidc)[:callback_url]
```

Comment:
I think we need to add this line here. Otherwise, I'm getting an error
```suggestion
  defp idp_redirect_uri("oidc_local"), do: Application.fetch_env!(:trento, :oidc)[:callback_url]
  defp idp_redirect_uri("oauth2_local"), do: Application.fetch_env!(:trento, :oauth2)[:callback_url]
```

The error is:
```
body: %{"error" => "invalid_grant", "error_description" => "Incorrect redirect_uri"}}
```

Reply:
Yeah thank you, this was eresad during the last rebase!